tests/end_to_end/test_kakarot.py

```diff
@@ -7,6 +7,7 @@
 from starknet_py.net.client_models import TransactionStatus
 from starknet_py.net.full_node_client import FullNodeClient
 
+from scripts.utils.starknet import fund_address
```

Comment:
sorry I misled you but plz put the import inside the fixture so that we can keep the test file the same when/if we change the fixture